free-2.md

```diff
@@ -46,6 +133,30 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+The ecosystem of formatting and typesetting is a rich one. Let's first consider
+the giants: [HTML](http://www.w3schools.com/html/) and
+[LaTeX](https://www.latex-project.org/).
+
+Each of these languages is complex, verbose, and full-feature. Anything that
+you want to format on the web can probably be managed by HTML. It includes
+support for multimedia and a host of other features. Its syntax is simple but
+results in long code. LaTeX is similar. It has syntax that is equally verbose
+but very general. The LaTeX macro system has even been shown to be Turing
+complete. Notably both these languages are whitespace insensitive.
+
+Now lets look at others, [Wiki](https://en.wikipedia.org/wiki/Help:Wiki_markup)
+and [Markdown](https://daringfireball.net/projects/markdown/). These languages
+are whitespace sensitive and aim to be much less verbose. They both choose to
+leave out features in order to simplify the language. Between their lack of
+features and relative new-ness they sometimes have unclear semantics -
+different interpreters interpret them differently to achieve slightly different
+goals (think of emoticons in GitHub's Markdown and the way in which each Wiki
+is just a little bit different).
+
+I think that Wiki and Markdown end up being much easier to compose, but limit
+user options a lot as a result. I think it would be a good idea to try to build
+a language which is as full-feature as LaTeX, but leans on whitespace to make
+some common patterns much cleaner to express.
```

Comment:
I like this.  Although thinking about it, it seems like a post-processor on your LaTeX could probably handle all of these issues.